apps/ui/src/views/Proposal.vue

```diff
@@ -310,7 +310,7 @@ watchEffect(() => {
                   <a
                     v-if="
                       votingPower?.status === 'success' &&
-                      votingPower.totalVotingPower === BigInt(0)
+                      votingPower.votingPowers.some(v => v.value > 0n)
```

Comment:
Wouldn't that show when only one strategies among n strategies have 0 VP, but user still have enough VP to vote ?

Reply:
Good catch!